index.js

```diff
@@ -105,8 +102,9 @@ const globFiles = async (patterns, {ignores, extensions, cwd}) => (
 const getConfig = async options => {
 	const {options: foundOptions, prettierOptions} = mergeWithFileConfig(normalizeOptions(options));
 	options = buildConfig(foundOptions, prettierOptions);
```

Comment:
Seems like you can just restructure directly here.

Reply:
Not sure what you meant, but did something